woocommerce/woocommerce
# Updates table styling and columns

### All Submissions:


After testing some extensions we found it could be confusing if a method had a name and a default title which was different. e.g. PayPal Hosted has a default title of `credit card` which showed in the table and made it hard to notice.

Fixed this by moving the method title into the first column.

This PR also improves enabled checkbox styling and the loading indicator which was blocking the entire table.

![woocommerce settings wordpress wordpress 2018-05-04 18-49-11](https://user-images.githubusercontent.com/90977/39643266-23207cb2-4fcc-11e8-9188-3b65b78c1b21.png)


### How to test the changes in this Pull Request:

1. View the Payments settings screen with variety of gateways enabled
2. Try toggling them on and off an ensure styling looks correct - there is no blockui

cc @laurendavissmith @avivapinchas
